language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java

```diff
@@ -0,0 +1,118 @@
+        switch (params.getCommand()) {
+            case CommandConstants.CMD_IMPORT_PACKAGE:
+                executeImportPackage(context);
+                break;
+            default:
+                // Do Nothing
+                break;
+                    
+        }
+    }
+
+    /**
+     * Execute the command, import package.
+     * @param context   Workspace service context
+     */
+    private static void executeImportPackage(WorkspaceServiceContext context) {
+        String documentUri = null;
+        
+        ApplyWorkspaceEditParams applyWorkspaceEditParams = new ApplyWorkspaceEditParams();
+        WorkspaceEdit workspaceEdit = new WorkspaceEdit();
+        VersionedTextDocumentIdentifier textDocumentIdentifier = new VersionedTextDocumentIdentifier();
+        
+        for (Object arg : context.get(ExecuteCommandKeys.COMMAND_ARGUMENTS_KEY)) {
+            if (((LinkedTreeMap) arg).get(ARG_KEY).equals(CommandConstants.ARG_KEY_DOC_URI)) {
+                documentUri = (String) ((LinkedTreeMap) arg).get(ARG_VALUE);
+                textDocumentIdentifier.setUri(documentUri);
+                context.put(DocumentServiceKeys.FILE_URI_KEY, documentUri);
+            } else if (((LinkedTreeMap) arg).get(ARG_KEY).equals(CommandConstants.ARG_KEY_PKG_NAME)) {
+                context.put(ExecuteCommandKeys.PKG_NAME_KEY, (String) ((LinkedTreeMap) arg).get(ARG_VALUE));
+            }
+        }
+        
+        if (documentUri != null && context.get(ExecuteCommandKeys.PKG_NAME_KEY) != null) {
+            String fileContent = context.get(ExecuteCommandKeys.DOCUMENT_MANAGER_KEY)
+                        .getFileContent(Paths.get(URI.create(documentUri)));
+            String[] contentComponents = fileContent.split(";");
+            int totalLines = fileContent.split("\\n|\\r\\n|\\r").length;
+            int lastNewLineCharIndex = Math.max(fileContent.lastIndexOf("\n"), fileContent.lastIndexOf("\r"));
+            int lastCharCol = fileContent.substring(lastNewLineCharIndex + 1).length();
+            BLangPackage bLangPackage = TextDocumentServiceUtil.getBLangPackage(context,
+                    context.get(ExecuteCommandKeys.DOCUMENT_MANAGER_KEY));
+            
+            String pkgName = context.get(ExecuteCommandKeys.PKG_NAME_KEY);
+            BLangImportPackage lastImport = bLangPackage.getImports().get(bLangPackage.getImports().size() - 1);
+            DiagnosticPos pos = lastImport.getPosition();
+            int endCol = pos.getEndColumn() - 1;
+            int endLine = pos.getEndLine() - 1;
+            int contentLengthToLastImport = 
+                    fileContent.substring(0, fileContent.indexOf(contentComponents[endLine])).length()
+                            + contentComponents[endLine].length() + 1;
+            String remainingTextToReplace = fileContent.substring(contentLengthToLastImport);
+            String editText = "\nimport " + pkgName + ";"
```

Comment:
"\n" will be a problem in windows? can we replace \n with \r\n?

Reply:
Fixed with 62b1ab43e8ea5ad94fc1a6fc1c504fa6538b49ef